.github/workflows/osx.yml

```diff
@@ -0,0 +1,39 @@
+name: osx
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [ "master" ]
+    tags: [ "*" ]
+
+jobs:
+  osx:
+    strategy:
+      fail-fast: false
+      matrix:
+        runs-on:
+          - macos-11
+          - macos-12
+        mode:
+          - Debug
+          - Release
+
+    runs-on: ${{ matrix.runs-on }}
+
+    steps:
+      - name: Clone the connector
+        uses: actions/checkout@v3
+
+      - name: Setup stable tarantool from brew
+        run: brew install tarantool
+
+      - name: build
+        run: |
+          mkdir build
+          cd build
+          cmake -DCMAKE_BUILD_TYPE=${{ matrix.mode }} -DTNTCXX_BUILD_TESTING=ON -DTNTCXX_ENABLE_SSL=OFF .. 
```

Comment:
> `-DTNTCXX_ENABLE_SSL=OF`

What about testing SSL? I see you already commited the same thing for ubuntu though, but would still be nice to track it in a separate ticket, if you are not planning to bring it right now. Anyways, I would still factor this out into the option matrix.

Reply:
Ok, I factored the option out and filed an issue about SSL.